lib/matplotlib/widgets.py

```diff
@@ -1934,8 +1934,15 @@ def onmove(self, event):
             self.linev.set_visible(False)
             self.lineh.set_visible(False)
             if self.needclear:
+                if self.useblit and self.background is not None:
+                    self.canvas.restore_region(self.background)
+                else:
+                    self.canvas.draw()
+                    self.canvas.restore_region(self.background)
+                    self.canvas.blit(self.ax.bbox)
+            else:
                 self.canvas.draw()
```

Comment:
I think this will get called repeatedly when moving around outside the axes, since we set `needclear` to `False` when we first moved outside.  I think the aim of the issue was to _not_ call this unless we have to.